core/src/main/java/com/adobe/aem/commons/assetshare/content/renditions/download/impl/AssetRenditionsDownloadServlet.java

```diff
@@ -78,20 +80,28 @@
 
     private static final String DEFAULT_ASSET_RENDITIONS_DOWNLOAD_ORCHESTRATOR = AssetRenditionsZipperImpl.class.getName();
 
+    @Reference(target="(distribution=classic)")
```

Comment:
@davidjgonzalez just so I'm reading this right, we expect that `AssetRenditionsDownloadServlet` is **only** available on `classic` distribution and that `AsyncAssetRenditionsDownloadServlet.java` will **only** be enabled on the cloud distribution. Therefore the servlets can have the same `sling.servlet.selectors` and no real danger of collision?

Reply:
Correct! Especially since neither should ever be used in the "other" distribution this prevents any goofiness turning one or the other on and breaking stuff